src/RenderWebGL.js

```diff
@@ -851,7 +853,7 @@ class RenderWebGL extends EventEmitter {
                 for (let d = candidateIDs.length - 1; d >= 0; d--) {
                     const id = candidateIDs[d];
                     const drawable = this._allDrawables[id];
-                    if (drawable.isTouching(worldPos)) {
+                    if (drawable.isTouching(worldPos, candidateIDsWereProvided)) {
```

Comment:
this assumption doesn't seem correct?  if we provide candidates we ignore visibility?

Reply:
If we provide candidateIDs, we ignore *invisibility* e.g. isTouching does not early exit if the tester is invisible